packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1442,6 +1485,8 @@ export class ContainerRuntime
 				disableChunking,
 				disableAttachReorder: this.disableAttachReorder,
 				disablePartialFlush,
+				idCompressorEnabled: compressorEnabled ? true : undefined,
+				abnormalAckRecoveryMethod: this.summaryStateUpdateMethod,
```

Comment:
This is not related to your change. Maybe it got added during rebase?

Reply:
I think this came from a merge, should be fixed now.